src/Adapter/MySQL.php

```diff
@@ -93,23 +93,26 @@ public function execute()
      */
     public function getQuery()
     {
+        // Prepare mapping for joined tables
         $filterToTableMapping = $this->getFieldMapping();
+
+        // Process and generate all fields for the SQL query below
         $orderField = $this->computeOrderByField($filterToTableMapping);
+        $selectFields = $this->computeSelectFields($filterToTableMapping);
```

Comment:
This is just a cosmetic change that has no effect, but the computations should be together.